src/lib/gas-refund-api.ts

```diff
@@ -174,20 +174,10 @@ export class GasRefundApi {
     };
   }
 
-  async getAllEntriesForEpoch(epoch: number): Promise<GasRefundParticipation[]> {
+  async getAllEntriesForEpoch(
+    epoch: number,
+  ): Promise<GasRefundParticipation[]> {
     const grpData = await GasRefundParticipation.findAll({
```

Comment:
Why do you know longer pick fields? You use everything or to just streamline the getter?

Reply:
this is purely used for debugging will be shooted